eq-author-api/tests/utils/contextBuilder/section/moveSection.js

```diff
@@ -77,8 +77,12 @@ const moveSectionMutation = `
       sections {
         id
         title
-        pageDescription
-        displayName
```

Comment:
Should this be removed?

Reply:
`pageDescription` - removed to be replaced with `sectionSummaryPageDescription`

`displayName` - will re-add this value